app/src/main/java/controller/Controller.java

```diff
@@ -1,12 +1,23 @@
 package controller;
 
+
+import controller.customerController.CustomersController;
+import controller.elevatorSystemController.ElevatorSystemController;
 import lombok.Setter;
 import model.Model;
 import java.util.concurrent.TimeUnit;
 
+/**
+ * Main controller .
+ *
+ * @see CustomersController
+ * @see ElevatorSystemController
+ */
 public class Controller {
+    public final ElevatorSystemController ELEVATOR_SYSTEM_CONTROLLER;
     public final Model MODEL;
 
+    private final CustomersController CUSTOMER_CONTROLLER;
     private final int TPS = 50;
```

Comment:
posortowac to aby kontroller byl obok kontrollera

Reply:
Ok